src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Tags/RenderBodyTag.cs

```diff
@@ -4,14 +4,13 @@
 using System.Threading.Tasks;
 using Fluid;
 using Fluid.Ast;
-using Fluid.Tags;
 using Microsoft.AspNetCore.Html;
 
 namespace OrchardCore.DisplayManagement.Liquid.Tags
 {
-    public class RenderBodyTag : SimpleTag
+    public class RenderBodyTag
     {
-        public override async ValueTask<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context)
+        public static async ValueTask<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context)
         {
             if (!context.AmbientValues.TryGetValue("ThemeLayout", out dynamic layout))
             {
```

Comment:
static methods could be inline candidates when there's no throw clause (would require ThrowHelper), I think this is too large though to be considered, but generally tThrowHelper is a good thing